.gitlab/functional_test/security_agent.yml

```diff
@@ -174,9 +174,8 @@ kitchen_stress_security_agent:
 
 kitchen_test_security_agent_windows_x64:
   extends:
-    - .kitchen_os_windows
     - .kitchen_test_security_agent
-    - .kitchen_azure_x64
+    - .kitchen_os_windows
```

Comment:
`kitchen_azure_x64` is [included](https://github.com/DataDog/datadog-agent/blob/nschweitzer/junit_still/.gitlab/kitchen_testing/windows.yml#L13) in `kitchen_os_windows`